app/helpers/mailer_helper.rb

```diff
@@ -176,6 +180,30 @@ def style_work_tag_metadata(tags)
     "#{label}#{style_work_tag_metadata_list(tags)}".html_safe
   end
 
+  def commenter_pseud_or_name_link(comment)
+    if comment.comment_owner
```

Comment:
Can we reduce the nested ifs by using `if comment.comment_owner.nil?` as a guard clause?